README.md

```diff
@@ -1,8 +1,8 @@
 # Welcome to The Odin Project Curriculum
 
-The Odin Project (also known as TOP) is an open-source community for learning full-stack web development. Our mission is to provide a comprehensive curriculum to learn web development for free. We help our students learn the skills and build the impressive portfolio of projects they need to get hired as a web developer.
+The Odin Project (also known as TOP) is an open-source community for learning full-stack web development. Our mission is to provide a comprehensive curriculum to learn web development for free. We help our students learn the skills and build an impressive portfolio of projects they need to get hired as a web developer.
```

Comment:
This was correct before: `the impressive portfolio` goes with `the skills` that you'll `need to get hired`.